modules/installation-and-upgrade/pages/container-deployment/snippet-warn-images-sl-micro.adoc

```diff
@@ -0,0 +1,5 @@
+[IMPORTANT]
+====
+Customized {productname} {productnumber} VM images are provided only for {sl-micro} {microversion}.
```

Comment:
Just to make sure, this is only valid for MLM and not for uyuni.

Reply:
ok. ATM, only mlm files include this snippet, but for safety I now moved it to the `mlm` dir